LICENSE

```diff
@@ -1,21 +1,674 @@
-MIT License
-
-Copyright (c) 2019 FOSS@Amrita
```

Comment:
If you even making it as GNUv3 you cannot remove
`Copyright (c) 2019 FOSS@Amrita`

Reply:
now fixed